mix.exs

```diff
@@ -1,10 +1,12 @@
 defmodule AeMdw.MixProject do
   use Mix.Project
 
+  @version "1.48.1"
+
   def project() do
     [
       app: :ae_mdw,
-      version: "1.48.1",
+      version: @version,
```

Comment:
I switched to @version because it's now used twice in the file. Will the release please task find and replace multiple instances? Will it replace the new one?

Reply:
Will the CI replace both places "1.48.1" will be hard coded if I remove this change to use a module attribute? The two places are here, and on line 123 of this file?

I can hard code 1.48.1 in both places, but it seems like a much worse solution, would quite likely break the CI anyway.

Reply:
I haven't tried it before but I think it's unlikely that it replaces in both places. One way to avoid the conflict is to pipe the list and and append the `releases`.